framework/source/class/qx/log/appender/Console.js

```diff
@@ -172,7 +172,8 @@ qx.Class.define("qx.log.appender.Console",
     process : function(entry)
     {
       // Append new content
-      this.__log.appendChild(qx.log.appender.Util.toHtml(entry));
+      var formatter = qx.log.appender.Formatter.getDefaultFormatter();
```

Comment:
If the goal here is to be able to use different formatters, then why would we always retrieve the _default_ formatter? Should this instead be
```
var formatter = qx.log.appender.Formatter.getFormatter();
```
and the `formatter` property just has a default value, but it can be changed?